assets/README.powershellcommunity.md

```diff
@@ -0,0 +1,91 @@
+* **[Oracle Linux][oracle-linux-uri]**
+  * Tags: `oraclelinux-7.5` , `6.1.1-oraclelinux-7.5`
+  * Dockerfile: [/release/community-stable/oraclelinux/docker/Dockerfile][oracle-linux-stable-dockerfile]
+  * Example: `docker pull pshorg/powershellcommunity:oraclelinux-7.5`
+* **[Parrot Linux][parrotsec-linux-uri]**
+  * Tags: `parrot-4.2.2` , `6.1.1-parrot-4.2.2`
+  * Dockerfile: [/release/community-stable/parrot/docker/Dockerfile][parrotsec-linux-stable-dockerfile]
+  * Example: `docker pull pshorg/powershellcommunity:parrot-4.2.2`
+
+[amazon-linux-stable-dockerfile]: https://github.com/PowerShell/PowerShell-Docker/blob/master/release/community-stable/amazonlinux/docker/Dockerfile
+[arch-linux-stable-dockerfile]: https://github.com/PowerShell/PowerShell-Docker/blob/master/release/community-stable/archlinux/docker/Dockerfile
+[blackarch-linux-stable-dockerfile]: https://github.com/PowerShell/PowerShell-Docker/blob/master/release/community-stable/blackarch/docker/Dockerfile
+[clear-linux-stable-dockerfile]: https://github.com/PowerShell/PowerShell-Docker/blob/master/release/community-stable/clearlinux/docker/Dockerfile
+[kali-linux-stable-dockerfile]: https://github.com/PowerShell/PowerShell-Docker/blob/master/release/community-stable/kali-rolling/docker/Dockerfile
+[oracle-linux-stable-dockerfile]: https://github.com/PowerShell/PowerShell-Docker/blob/master/release/community-stable/oraclelinux/docker/Dockerfile
+[parrotsec-linux-stable-dockerfile]: https://github.com/PowerShell/PowerShell-Docker/blob/master/release/community-stable/parrot/docker/Dockerfile
+
+[amazon-linux-uri]: https://aws.amazon.com/amazon-linux-2/
+[arch-linux-uri]: https://www.archlinux.org/
+[blackarch-linux-uri]: https://www.blackarch.org/
+[clear-linux-uri]: https://www.clearlinux.org/
+[kali-linux-uri]: https://www.kali.org/
+[oracle-linux-uri]: https://www.oracle.com/linux/
+[parrotsec-linux-uri]: https://www.parrotsec.org/
+
+## Legal and licensing
+
+PowerShell is licensed under the [MIT license][].
+
+[MIT license]: https://github.com/PowerShell/PowerShell/tree/master/LICENSE.txt
+
+[Third-Party Software Notices and Information](https://github.com/PowerShell/PowerShell/blob/master/ThirdPartyNotices.txt)
+
+## About the image
+
+PowerShell Core is a cross-platform (Windows, Linux, and macOS) automation and configuration tool/framework. It works well with your existing tools and is optimized for dealing with structured data (for example, JSON, CSV, and XML), REST APIs, and object models.
+It includes a command-line shell, an associated scripting language, and a framework for processing cmdlets.
+
+If you are new to PowerShell and want to learn more, see the [getting started][] documentation.
+
+[getting started]: https://github.com/PowerShell/PowerShell/tree/master/docs/learning-powershell
+
+## How to use this image
+
+See our [Docker examples](https://github.com/PowerShell/PowerShell/tree/master/docker#examples).
```

Comment:
This has been moved to this repo (needs to be fixed in the other readme as well)

Reply:
Do we have an existing markdown file some where that needs to be moved into this repo? Or do we need to make one in this repo and update the link in both the `README.powershell.md` and README.powershellcommunity.md` later.

Reply:
I updated the link to the file in this repo